raiden/ui/cli.py

```diff
@@ -493,3 +497,26 @@ def append_report(subject, data):
     else:
         print('[5/5] smoketest had errors, report was written to {}'.format(report_file))
         sys.exit(1)
+
+
+@run.command()
+@OPTION_ADDRESS
+@OPTION_DATADIR
+def removedb(address, datadir):
+    """
+    Delete local cache and database of this address or all if none is specified.
+    """
+    import shutil
+
+    address_hex = address_encoder(address)
+    user_db_dir = os.path.join(datadir, address_hex[:8]) if address_hex else datadir
```

Comment:
May I suggest making some simple tests to check if `datadir` is really a raiden database? Maybe being sure it only contains subdirectories starting with `0x`, just so we don't remove user homedir if he mistype something (even with prompt below)